lib/rucio/api/did.py

```diff
@@ -530,3 +531,67 @@ def list_archive_content(scope, name):
     dids = did.list_archive_content(scope=scope, name=name)
     for d in dids:
         yield api_update_return_dict(d)
+
+
+def add_did_to_followed(scope, name, account, session=None):
```

Comment:
Please change all scopes and accounts to `InternalScope` and `InternalAccount`. E.g. have a look on the `create_did_sample` method in this module.